core/src/main/java/com/graphhopper/reader/InvalidIdException.java

```diff
@@ -0,0 +1,11 @@
+package com.graphhopper.reader;
+
+public class InvalidIdException extends RuntimeException{
```

Comment:
Let's just use `IllegalArgumentException` here? We did not really start using a whole lot of different exceptions so far.